Build docker-localnode failed - LocalCluster

Before running this i ran `make build linux `in the root directory.
and then i ran this:
`Make build-docker-localnode
`
After running for 15-20 mins it gave me this error:


```
E[21126-12-21|11:49:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:50:16.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:50:27.221] Failed to write ping                         module=rpc-server protocol=websocket remote=127.0.0.1:47284 err="write tcp 127.0.0.1:18771->127.0.0.1:47284: write: broken pipe"
E[21126-12-21|11:50:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:51:17.235] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:51:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:52:16.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:52:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:53:16.983] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:53:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:54:16.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:54:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:55:16.991] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:55:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:56:16.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:56:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:57:16.895] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:57:46.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:58:16.863] Couldn't connect to any seeds                module=p2p 
E[21126-12-21|11:58:46.863] Couldn't connect to any seeds                module=p2p 
SIGQUIT: quit

```

```
Makefile:222: recipe for target 'test' failed
make[1]: *** [test] Error 1
make[1]: Leaving directory '/home/kunaldhariwal/go/src/github.com/tendermint/tendermint'
Makefile:295: recipe for target 'build-docker-localnode' failed
make: *** [build-docker-localnode] Error 2
```

`FAIL	github.com/tendermint/tendermint/rpc/client	600.013s`
also same thing came for consensus.

Apart from all this it ran multiple tests and stuff, which is usual i guess. Resulted output as **?** for some, **ok** for some and **fail** for very few.

--- a/Makefile
+++ b/Makefile
@@ -292,9 +292,7 @@ build-linux:
 	GOOS=linux GOARCH=amd64 $(MAKE) build
 
 build-docker-localnode:
-	cd networks/local
-	make
-	cd -
+	@cd networks/local && make
 
 # Run a 4-node testnet locally
 localnet-start: localnet-stop

